src/django/api/models/moderation_event.py

```diff
@@ -0,0 +1,117 @@
+import uuid
+from django.db import models
+from api.models.facility.facility_claim import FacilityClaim
+
+
+class ModerationEvent(models.Model):
+    '''
+    Data that is needed for moderation queue.
+    '''
+    uuid = models.UUIDField(
+        default=uuid.uuid4,
+        editable=False,
+        unique=True,
+        db_index=True,
+        help_text=(
+            'Unique identifier to make moderation '
+            'event table more reusable across the app.'
+        )
+    )
+
+    created_at = models.DateTimeField(
+        auto_now_add=True,
+        help_text='Date when the moderation queue entry was created.'
+    )
+
+    updated_at = models.DateTimeField(
+        auto_now=True,
+        help_text='Date when the moderation queue entry was last updated.',
+        db_index=True
+    )
+
+    status_change_date = models.DateTimeField(
+        null=True,
+        help_text='Date when the moderation decision was made.'
+    )
+
+    claim = models.OneToOneField(
```

Comment:
1. Should we connect `contributor_id` with current table as we will show Contributor data in Moderation Queue?
2. In design https://docs.google.com/presentation/d/1ZMXCwW2JxpXVS-MEtnLJTlcWCrkFGTub0XSUfWZhCPQ/edit#slide=id.g2ef9662a60c_0_124 there is a `match_status` field that will shown for moderators in Moderation Queue page (N/A, New Facility, Matched). I just want to double check do we need this in our MVP? As after moderator will do some action inside Contirbutor Record like "create new facility" or "matched to potential" this status will be updated.
cc. @VadimKovalenkoSNF @protsack-stephan

Reply:
Yes, good point. Thanks for noting!

Reply:
I suppose, yes. Contributor id should be there as well. Thanks for noticing this, Roman.

Reply:
> 2

From what I understood from a prevs API Tech sessions, we can omit anything around potential matches and hence, `match_status` field is not needed here.

Reply:
@VadimKovalenkoSNF I suppose `match_status` could be optional for MVP at least.
cc @protsack-stephan

Reply:
> I suppose match_status could be optional for MVP at least.

I don’t believe we need it, but I'm open to discussing it further.

Reply:
I don't think we need match status at the moment, whether we need it at all is a separate discussion.